### PR TITLE
[FEATURE] notifications using expires, status, entities and notif_attrs

### PR DESCRIPTION
**Pull request by iariasleon**
```
                    SUMMARY:
-------------------------------------------------
  - components\ngsiv2\subscriptions\notifications\notifications_expiration_and_status.feature >> passed: 6, failed: 0, skipped: 0 and total: 6 with duration: 9.063 seconds.
  - components\ngsiv2\subscriptions\notifications\notifications_notification_attrs.feature >> passed: 19, failed: 0, skipped: 0 and total: 19 with duration: 25.150 seconds.
  - components\ngsiv2\subscriptions\notifications\notifications_subject_entities.feature >> passed: 50, failed: 0, skipped: 0 and total: 50 with duration: 70.329 seconds.
-------------------------------------------------
  Date: Friday, August 26, 2016 15:19:18
-------------------------------------------------
3 features passed, 0 failed, 0 skipped
75 scenarios passed, 0 failed, 0 skipped
676 steps passed, 0 failed, 0 skipped, 0 undefined
Took 1m44.542s
```
@kzangeli @fgalan @crbrox 

Condition and throttling still are pending

### CODE REVIEW

**Comment by jcanonav** [comment]
LGTM

File: test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_expiration_and_status.feature [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: verify notifications from subscriptions with diferent formats using NGSIv2
+  As a context broker user
+  I would like toverify notifications from subscriptions with diferent formats using NGSIv2
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_expiration_and_status.feature]: toverify -> to verify

File: test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_subject_entities.feature [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: verify notifications from subscriptions with diferent formats using NGSIv2
+  As a context broker user
+  I would like toverify notifications from subscriptions with diferent formats using NGSIv2
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_subject_entities.feature]: toverify -> to verify

(Second time I spot this... please do a global text search to locate all the case)

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_subject_entities.feature]: Fixed in  	dc82588

File: test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_expiration_and_status.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_expiration_and_status.feature]: Fixed in  	dc82588

**Comment by fgalan** [comment]
LGTM

**Pull request merged**